docs/cn/quickstart.md

```diff
@@ -0,0 +1,145 @@
+
+# QuickStart
+本教程会涵盖使用OpemMLDB构建机器学习应用的整个流程，包括离线特征抽取、模型训练，以及在线的数据导入、在线特征抽取、模型预测等步骤。希望通过该教程，读者可以了解通过OpenMLDB，如何快速完成从原始数据处理到模型上线的整个生命周期。
+
+
+OpenMLDB提供了Java、C++、Python APIs，该示例中，我们使用Python API。
+
+为了方便理解，我们会基于Kaggle比赛[Predict Taxi Tour Duration数据集](https://raw.githubusercontent.com/4paradigm/OpenMLDB/main/demo/predict-taxi-trip-duration-nb/demo/data)，来介绍整个流程。数据集和相关代码可以在[这里](https://github.com/4paradigm/OpenMLDB/tree/main/demo/predict-taxi-trip-duration-nb/demo)查看，并且可以根据步骤尝试运行。
+
+## 离线
+### 特征抽取
+对于特征抽取，用户首先需要通过对数据集的了解，构建特征抽取的SQL脚本，例如，在Taxi Tour Duration数据集中，我们可以构建如下特征抽取的[SQL](https://github.com/4paradigm/OpenMLDB/blob/main/demo/predict-taxi-trip-duration-nb/demo/fe.sql):
+```sql
+select trip_duration, passenger_count,
+sum(pickup_latitude) over w as vendor_sum_pl,
+max(pickup_latitude) over w as vendor_max_pl,
+min(pickup_latitude) over w as vendor_min_pl,
+avg(pickup_latitude) over w as vendor_avg_pl,
+sum(pickup_latitude) over w2 as pc_sum_pl,
+max(pickup_latitude) over w2 as pc_max_pl,
+min(pickup_latitude) over w2 as pc_min_pl,
+avg(pickup_latitude) over w2 as pc_avg_pl ,
+count(vendor_id) over w2 as pc_cnt,
+count(vendor_id) over w as vendor_cnt
+from t1
+window w as (partition by vendor_id order by pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW),
+w2 as (partition by passenger_count order by pickup_datetime ROWS_RANGE BETWEEN 1d PRECEDING AND CURRENT ROW);
+```
+
+
+通过执行特征抽取的SQL，可以从原数据里提取出我们需要的特征数据集，用来进行训练。
+
+对于离线特征抽取的输入数据，可以直接存放在本地文件或者HDFS上面，然后通过调用spark sql来运行特征抽取。
+
+
+Python示例代码如下（完整代码参考[这里](https://github.com/4paradigm/OpenMLDB/blob/main/demo/predict-taxi-trip-duration-nb/demo/openmldb_batch.py)）：
+
+```python
+def run_batch_sql(sql):
+    spark = SparkSession.builder.appName("OpenMLDB Demo").getOrCreate()
+    parquet_predict = "./data/taxi_tour_table_predict_simple.snappy.parquet"
+    parquet_train = "./data/taxi_tour_table_train_simple.snappy.parquet"
+    train = spark.read.parquet(parquet_train)
+    train.createOrReplaceTempView("t1")
+    train_df = spark.sql(sql)
+    train_set = train_df.toPandas()
+    predict = spark.read.parquet(parquet_predict)
+    predict.createOrReplaceTempView("t1")
+    predict_df = spark.sql(sql)
+    predict_set = predict_df.toPandas()
+    return train_set, predict_set
+```
+
+
+***注意***：需要使用[OpenMLDB Spark Distribution](https://github.com/4paradigm/spark)
```

Comment:
you may also refer users to this doc about how to compile and config our spark distribution
https://github.com/4paradigm/OpenMLDB/blob/main/docs/en/compile.md

Reply:
the second section "Optimized Spark Distribution for OpenMLDB (Optional)"